pkg/registry/core/service/storage/rest_test.go

```diff
@@ -46,28 +52,129 @@ import (
 // It is now testing mostly the same things as other resources but
 // in a completely different way. We should unify it.
 
+type serviceStorage struct {
+	GottenID           string
+	UpdatedID          string
+	CreatedID          string
+	DeletedID          string
+	Created            bool
+	DeletedImmediately bool
+	Service            *api.Service
+	OldService         *api.Service
+	ServiceList        *api.ServiceList
+	Err                error
+
+	mu sync.Mutex
```

Comment:
what does it protect? please put on-top of the block it belongs to.

Reply:
Was leftover from a much earlier version of this test (this file was moved, but then refactored because it talked to a registry instead of to a rest storage).  I'm going to remove it, no threaded operations appear to be performed anywhere in here.